source/shared/core_sqlsrv.h

```diff
@@ -997,6 +1045,8 @@ struct sqlsrv_conn : public sqlsrv_context {
     sqlsrv_conn( SQLHANDLE h, error_callback e, void* drv, SQLSRV_ENCODING encoding  TSRMLS_DC ) :
         sqlsrv_context( h, SQL_HANDLE_DBC, e, drv, encoding )
     {
+        server_version = SERVER_VERSION_UNKNOWN;
+        driver_version = ODBC_DRIVER_13; 
```

Comment:
the default is 13, later in the code if 13 hasn't been found the it sets it to 11